auth-foundation/src/main/java/com/okta/authfoundation/credential/SecurityOptions.kt

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2024-Present Okta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.okta.authfoundation.credential
+
+import com.okta.authfoundation.AuthFoundationDefaults
+import kotlinx.serialization.Serializable
+
+@Serializable
+sealed interface SecurityOptions {
```

Comment:
I defined this within the Credential namespace as `Credential.Security`. Is it possible to do the same here, for consistency?

Reply:
Sure, I will do this